src/index.js

```diff
@@ -38,8 +39,8 @@ export default function mitt(all: EventHandlerMap) {
 		 * @memberOf mitt
 		 */
 		off(type: string, handler: EventHandler) {
-			let e = all[type] || (all[type] = []);
```

Comment:
the assignment isn't needed, just it produced a smaller output size because it is duplicated and thus cheap when gzipped. I'd be curious to see the size with & without this change.